.github/workflows/check-code-and-run-tests.yml

```
@@ -0,0 +1,31 @@
+name: Check code and run tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04  # Python 3.6 is no longer available on ubuntu_latest
```
actions/setup-python#355